core-rust/state-manager/src/state_computer.rs

```diff
@@ -97,18 +97,6 @@ use std::ops::Deref;
 use std::sync::Arc;
 use std::time::{Instant, SystemTime};
 
-#[derive(Debug, Categorize, Encode, Decode, Clone, Default)]
-pub struct LoggingConfig {
-    pub engine_trace: bool,
-    pub state_manager_config: StateComputerLoggingConfig,
-}
-
-// TODO: Replace this with better loglevel integration
```

Comment:
_(note from author)_

I believe "better loglevel integration" is exactly what I do in this PR; so I now simply log the rejected transaction at DEBUG (instead of using this flag).